### PR TITLE
Readme and dual licensing

### LINKED ISSUE #5: Change license to a free culture license
### Describe what feature you'd like. Pseudo-code, mockups, or screenshots of similar solutions are encouraged!

I'd like to relicense this repo under CC-BY-SA. There is no IP to speak of, and we are making it easier for other users to reuse the template and its derivatives in other contexts.
- [x] @mtwente needs to approve the relicensing.

### What type of pull request would this be?

Other

### Any links to similar examples or other references we should review?

- https://creativecommons.org/public-domain/freeworks/

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

## Proposed changes

- Improve `README.md`
- Fix #5

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **Documentation**
  - Updated licensing to GNU Affero General Public License version 3 for better community engagement in network server software.
  - Transitioned to "Attribution 4.0 International" license, broadening the scope of permissible uses of the content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,14 +104,26 @@ The structure of this repository is designed to be as simple as possible. The ma
...
+Previewing your document ensures it looks correct before submission. Here is how you can preview your abstract using the Quarto CLI:
+
+1. **Install Quarto:**
+   - Follow the detailed installation instructions on the [Quarto website](https://quarto.org/docs/get-started/).
+
+2. **Open Your Terminal:**
+   - Access your terminal or command prompt. On Windows, you can search for "cmd" or "Command Prompt" in your start menu. On macOS, open "Terminal" from your Applications/Utilities folder.
+
+3. **Navigate to Your Repository:**
+   - Use the `cd` command to change directories to the root directory of your repository, where your `index.qmd` file is located.
```
Comment (maehr) [inline README.md:118]: Should be root of repo, right?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,90 @@ Welcome to the official repository for the extended abstracts of the Digital His
...
+1. **Create a GitHub Account:**
+   - If you do not already have a GitHub account, create one [here](https://github.com/signup) for free.
+
+2. **Create Your Repository:**
+   - Click the ["Use this template"](https://github.com/new?template_name=abstract-template&template_owner=digihistch24) button to start a new repository based on this template.
+   - Enter a name for your repository and decide whether to make it public or private.
+   - Click "Create repository from template" to set up your new project.
+
+3. **Access and Edit Your Abstract:**
+   - In your new repository, navigate to the `abstract-template` folder.
```
Comment (mtwente) [inline README.md:29]:
```suggestion
   - In your new repository, navigate to the `abstract-template` folder and give it another name, e.g. your (shortened) submission's title.
```

File: README.md (hunk 1 above)
Comment (mtwente) [reply README.md:118]:
```suggestion
   - Use the `cd` command to change directories to the directory of your repository where your `index.qmd` file is located.
```

Comment (mtwente) [reply README.md:118]:
```suggestion
   - Use the `cd` command to change directories to the directory of your repository where your `index.qmd` file is located.
```

Comment (mtwente) [inline README.md:118]:
```suggestion
   - Use the `cd` command to change directories to the directory of your repository where your `index.qmd` file is located.
```

Comment (mtwente) [reply README.md:118]: ATM it is, but not necessarily. See suggested change.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,90 @@ Welcome to the official repository for the extended abstracts of the Digital His
...
+   - After making your edits, scroll to the bottom of the page.
+   - Enter a brief description of the changes you made in the "Commit changes" box.
+   - Click "Commit changes" to save your edits.
+
+5. **Preview and Collaborate:**
+   - To preview your abstract, follow the steps in the "Previewing the Abstract" section below.
+   - To collaborate, go to the "Settings" tab in your repository, choose "Manage access," and then "Invite a collaborator."
+
+6. **Submit Your Abstract:**
+   - When your abstract is ready for submission, download the repository as a ZIP file.
```
Comment (mtwente) [inline README.md:43]: Do we want the zip file as standard/preferred mode of submission? I think we should only suggest this as an alternative method. Won't we just end up with lots of emails with zip folders when somebody decides to update something afterwards?

**Review by mtwente — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (maehr) [reply README.md:29]: Not really necessary IMHO

File: README.md [hunk 4]
```diff
@@ -9,7 +9,90 @@ Welcome to the official repository for the extended abstracts of the Digital His
...
+1. **Create a GitHub Account:**
+   - If you do not already have a GitHub account, create one [here](https://github.com/signup) for free.
+
+2. **Create Your Repository:**
+   - Click the ["Use this template"](https://github.com/new?template_name=abstract-template&template_owner=digihistch24) button to start a new repository based on this template.
+   - Enter a name for your repository and decide whether to make it public or private.
+   - Click "Create repository from template" to set up your new project.
+
+3. **Access and Edit Your Abstract:**
+   - In your new repository, navigate to the `abstract-template` folder.
```
[the remark this answers is not in the document]
Comment (maehr) [reply README.md:29]: @mtwente this is confusing, but I am hesitant to change the folder structure. Do you have a fix in mind?

[the remark this answers is not in the document]
Comment (mtwente) [reply README.md:29]: What about renaming the subfolder to `your-abstract`? Then the folder structure would be:
- abstract-template
  - .github/ISSUE_TEMPLATE
  - your-abstract
    - data
    - images
    - _quarto.yml
    - ...
  - .gitattributes
  - .gitignore
  - ...

**Review by mtwente — APPROVED** [review]

**Pull request merged**